README.md

```diff
@@ -17,16 +17,15 @@ Other than tracing, Tracee is also capable of capturing files written to disk or
 
 ### Prerequisites
 
-- To run, Tracee requires Linux kernel version >= 4.14
-
-Not required if using the Docker image:
-- C standard library (tested with glibc)
+- Linux kernel version >= 4.14
+- Relevant kernel headers available under `/usr/src`
```

Comment:
This is not true for arch (/manjaro), where the headers (not links) are located at /lib/modules/$(KERN_RELEASE)/build

Reply:
Actually, the location is not a requirement at all as it can be set by environment variables

Reply:
maybe I change this to :
Relevant kernel headers available under a conventional location?
?

Reply:
Let's not lose info - it was useful to me to be able to check that there was nothing in `/usr/src` and that was likely my problem. How about saying something like "Relevant kernel headers available under `/usr/src` for most distributions (`/lib/modules/$(KERN_RELEASE)/build` for arch / manjaro)" 

We also need to tell users how to fulfill this requirement. Even if we just say "For example, on Ubuntu, run `sudo apt-get install linux-headers-$(uname -r)`" that gives a hint for what you'd have to do on other distro's